src/sorcha/sorcha.py

```diff
@@ -244,10 +251,13 @@ def runLSSTSimulation(args, configs):
             + str(len(observations.index))
         )
 
-        verboselog("Randomising astrometry...")
-        observations = PPRandomizeMeasurements.randomizeAstrometry(
-            observations, args._rngs, sigName="astrometricSigma_deg", sigUnits="deg"
-        )
+        if configs["randomization_on"]:
+            verboselog("Randomizing astrometry...")
+            observations = PPRandomizeMeasurements.randomizeAstrometry(
+                observations, args._rngs, sigName="astrometricSigma_deg", sigUnits="deg"
```

Comment:
We want to make sure the columns added by randomize Astrometry are added I think but just the values are the same as the one calculated

Reply:
Would something like the below work in sorcha.py or would you prefer I did this inside the function:

```
        if configs["randomization_on"]:
            verboselog("Randomizing astrometry...")
            observations = PPRandomizeMeasurements.randomizeAstrometry(
                observations, args._rngs, sigName="astrometricSigma_deg", sigUnits="deg"
            )
        else:
            verboselog("Randomization turned off in config file. No astrometric randomization performed.")
            observations["RATrue_deg"] = observations["RA_deg"]
            observations["DecTrue_deg"] = observations["Dec_deg"]

```

Reply:
something like this but you might need to explicitly copy.

Reply:
Done.